### PR TITLE
feat : added sign up auth

### PR DESCRIPTION
**Pull request by rutvikpumak**
-> Basic signup auth added
-> go to the login page and click to "create account"
-> after filling all the fields, you are good to go
-> again click on user profile icon and check your info is coming right there or now
-> Validation not added for now

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
 } from "./pages";
 import Mockman from "mockman-js";
 import Navbar from "./component/Navbar/Navbar";
-import { Login } from "./pages/Auth";
+import { Login, Signup } from "./pages/Auth";
```
Comment (ScoobyDubeyDo) [inline src/App.js:13]: **suggestion**: Try implementing barrel export here for a better experience

File: src/pages/Auth/Login.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
 import { useNavigate } from "react-router";
```
Comment (ScoobyDubeyDo) [inline src/pages/Auth/Login.js]: **must**: Both of these can be imported from `react-router-dom`

File: src/pages/Auth/Login.js [hunk 3]
```diff
@@ -91,10 +92,12 @@ export function Login() {
             Login with Test Credentials
           </div>
 
-          <div className="auth-secondary-btn text-center">
-            Create New Account{" "}
-            <i className="fa fa-chevron-right" aria-hidden="true"></i>
-          </div>
+          <Link to="/signup">
+            <div className="auth-secondary-btn text-center">
+              Create New Account{" "}
```
Comment (ScoobyDubeyDo) [inline src/pages/Auth/Login.js]: suggestion: remove all the `{" "}` from your code, use padding or `&nbsp` if extra space is needed

File: src/pages/Auth/Signup.js [hunk 4]
```diff
@@ -1,61 +1,119 @@
...
-          <div class="auth-checkbox">
-            <label class="select-input">
+          <div className="auth-checkbox">
+            <label className="select-input">
               <input
                 type="checkbox"
                 name="light"
-                class="checkbox-input"
+                className="checkbox-input"
                 value=""
```
Comment (ScoobyDubeyDo) [inline src/pages/Auth/Signup.js]: **suggestion**: remove the `value` attribute if the component is not controlled

**Review by ScoobyDubeyDo — APPROVED** [review]
The code looks clean and almost perfect, I have suggested a few things.

**Review by Vikas2426 — APPROVED** [review]

**Comment by Vikas2426** [comment]
Aman has already suggested the required changes. Otherwise looks good !

**Pull request merged**